vignettes/articles/theme.Rmd

```diff
@@ -69,3 +100,5 @@ output:
       version: 4
 ---
 ```
+
+<!--Carson- can we provide even just one example rule here using a bootstrap variable?-->
```

Comment:
My intention was that the link above `#a-basic-themeable-component` would serve as the example, but I suppose that could be made more obvious?

Reply:
I'm stuck coming up with a basic example here, perhaps I'm not understanding how this should work? I have a custom `.scss` file that is read into the YAML.

Here is the file:

```scss
.bs-purple {
  background-color: $purple-900;
}

.penguin-purple {
  background-color: purple;
}
```

And in my dashboard I have:

```html
<p class="bs-purple">I'm bootstrap purple</p>
<p class="penguin-purple">I'm penguin purple</p>
```

When I knit I see:
<img width="1432" alt="Screen Shot 2021-04-03 at 4 05 11 PM" src="https://user-images.githubusercontent.com/12160301/113490161-637be000-9496-11eb-8f4d-e6ad47f1c6c9.png">

Reply:
Hmm, seems like that should work, send me a reprex? Btw, this should also be possible via the new bslib knitr engine

Reply:
Is there a simpler example? I'm trying to show a one-liner for this section: https://flexdashboard-pkg.netlify.app/articles/theme.html#using-css-or-sass-1

The link https://rstudio.github.io/bslib/articles/theming.html#a-basic-themeable-component left me a bit lost without using Shiny.

Here is my repo where I was testing: https://github.com/apreshill/bsrmd/blob/master/dash.Rmd

Reply:
Ahh, `flex_dashboard` wasn't passing `css` to `html_document_base()`. I just pushed a fix, try again with `remotes::install_github("rstudio/flexdashboard")`

Reply:
Same example but using ` flexdashboard   0.5.2.9000 2021-04-06 [1]` now neither background color work.
<img width="1438" alt="Screen Shot 2021-04-06 at 7 57 28 AM" src="https://user-images.githubusercontent.com/12160301/113707415-df9c4080-96ad-11eb-80fb-8a02554d01b4.png">

Reply:
Ack! Sorry, I did that too quickly without verifying to try to unblock you. I was thinking that `html_document_base` had a `css` argument, but turns out it doesn't, so I've added it in https://github.com/rstudio/rmarkdown/pull/2095 (feel free to try again with `remotes::install_github("rstudio/rmarkdown#2095")`)

Reply:
No problem! That did it 🎉 
<img width="1404" alt="Screen Shot 2021-04-06 at 9 11 18 AM" src="https://user-images.githubusercontent.com/12160301/113715957-11b2a000-96b8-11eb-8629-2199ac1e75e1.png">